assemblies/fuse-karaf-framework/src/main/filtered-resources/resources/etc/org.apache.karaf.features.xml

```diff
@@ -401,6 +407,7 @@
                 <f:bundle>mvn:org.jboss.fuse.modules/fuse-stax2-compatibility/${project.version}</f:bundle>
             </feature>
         </replacement>
+        <!-- This doesn't work -->
         <replacement mode="merge">
             <feature name="artemis-core-client" version="${version.org.apache.activemq.artemis}">
```

Comment:
@grgrzybek  this cannot work because these three bundles are needed by netty-core